packages/boilerplate-generator/boilerplate_web_template.js

```diff
@@ -0,0 +1,90 @@
+// Template function for rendering the boilerplate html
+// Replicates the template defined in boilerplate_web.browser.html
+// XXX Does not necessarily preserve formatting (e.g. additionalStaticJs newlines)
+// Arguments: root : { htmlAttributes, css : [{ url }], bundledJsCssUrlRewriteHook : Function, head, dynamicHead, body, dynamicBody, inlineScriptsAllowed, additionalStaticJs, meteorRuntimeConfig }
+
+Boilerplate_Web_Browser_Template = function(root) {
+  return [].concat(
+
+    // XXX is htmlAttributes ever anything but {}?
+    // may just be a generic Blaze/Spacebars thing.
+    function props1(htmlAttributes) {
+      return [
+        ['<html'].concat(Object.keys(htmlAttributes).map(function(key) {
+
+          // XXX probably need to wrap strings in "".
```

Comment:
Definitely want to have some escaping here! Not just wrapping, also doing proper encoding